tests/cmd/extract-list/extract-list.hifive1.0.stdout

```diff
@@ -0,0 +1,22 @@
+        SIZE NAME
```

Comment:
THis entire commit mostly looks like auto-generated files. So, I did not look closely. Let me know if there is any specific part of this commit that you want me to review

Reply:
Correct, all the test are auto generated.  I just modified one line in cli_test.rs.  I will split it up so it is clear what is auto generated.